update_collections.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+COLLECTIONS = {"all" => 2062901859,
+               "boia" => 271744772,
+               "frus" => 155520420,
+               "serial_set" => 148631352,
+               "epa" => 659388570,
+               "crc" => 1491645774,
+               "trail" => 1100951433}.freeze
+
+def base_command
+  who = `whoami`.chomp
+  "HT_DEV= MB_SUPERUSER=#{who} /htapps/babel/mb/scripts/batch-collection.pl -o feddocs@hathitrust.org "
+end
+
+def apply_additions(name:, id:)
```

Comment:
`apply_additions` and `apply_deletions` are very same-y but probably not worth DRY-ing out.

Reply:
Agreed.